primitives/src/common.rs

```diff
@@ -35,6 +37,20 @@ pub type PalletString = Vec<u8>;
 #[cfg(feature = "std")]
 pub type PalletString = String;
 
+pub type BoundedPalletString = BoundedVec<u8, ConstU32<256>>;
+
+pub trait FromStr: Sized {
+	type Err;
+	fn from_str(inp: &str) -> Result<Self, Self::Err>;
+}
```

Comment:
i couldnt implement it because it is in a different crate i think

Reply:
Yes, but there is a workaround, which you can implement because of some niche-feature of the orphan rule.

```rust
impl From<&str> for Blabla {}
```

This works because of the generic param of the `From` trait.

Reply:
Sorry, I mixed something up, you are right about that 👍